cmd/gcp-controller-manager/loops.go

```diff
@@ -56,7 +56,7 @@ func loops() map[string]func(*controllerContext) error {
 			return nil
 		},
 		"certificate-signer": func(ctx *controllerContext) error {
-			signer, err := newGKESigner(ctx)
+			signer, err := newGKESigner(ctx, ctx.client.CertificatesV1beta1().CertificateSigningRequests())
```

Comment:
by scoping this to certificates/v1beta1, this will need rework in 1.19 when the controller changed to work in terms of csr/v1 (e.g. https://github.com/kubernetes/kubernetes/pull/91713/files#diff-05476437bf221fbe4cab028bf7ffbb74L103)

I don't feel that strongly either way, but the kube-controller-manager version of this just passes in a client